packages/mobile/src/web3/saga.ts

```diff
@@ -372,7 +376,7 @@ export function* addAccountToWeb3Keystore(key: string, currentAccount: string, p
     )
     yield put(setAccountInWeb3Keystore(account))
   } catch (e) {
-    Logger.debug(TAG + '@addAccountToWeb3Keystore', 'Failed to import raw key')
+    Logger.debug(TAG + '@addAccountToWeb3Keystore', `Failed to import raw key: ${e}`)
```

Comment:
Should we use `.error` here which can take `e` as the 3rd param? It formats the exception messages nicely